lang/po/Firmware_cs.po

```diff
@@ -253,7 +253,7 @@ msgstr "Kopirovat vybrany jazyk?"
 # MSG_CRASHDETECT c=13
 #: messages.c:30
 msgid "Crash det."
-msgstr ""
+msgstr "Det. narazu."
```

Comment:
Why is the "." at the end in CZ? `narazu` is not a shorted word -> no need to add a `.`